.travis.yml

```diff
@@ -7,12 +7,9 @@ notifications:
   email: false
 node_js:
   - stable
-  - '0.10'
-before_install:
-  - npm i -g npm@^3.0.0
+  - '6'
```

Comment:
It would be best to test on all versions that ESLint supports, i.e. currently 4, 5, 6 & 7.